core/src/main/java/io/parsingdata/metal/data/Source.java

```diff
@@ -17,14 +17,16 @@
 package io.parsingdata.metal.data;
 
 import java.io.IOException;
+import java.math.BigInteger;
 
 public abstract class Source {
```

Comment:
Is it possible to turn this into an interface, and make the `slice()` method a `default` method?

Reply:
Excellent point, but we'll address this in #210 as well.